sdk/storage/azure-storage-datamovement/inc/azure/storage/datamovement/tasks/download_blob_directory_task.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "azure/storage/datamovement/task.hpp"
+#include "azure/storage/datamovement/blob_folder.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs { namespace _detail {
+
+  struct DownloadBlobDirectoryTask : public _internal::TaskBase
```

Comment:
I find this kind of strange. We dont have real directory in blob service. How do we call this in Track1 DMLib?

Reply:
We call it blob directory...

Reply:
Ok then. I don't have any better ideas. Let's go with blob directory.